nanshe_ipython.ipynb

```diff
@@ -1074,14 +1078,16 @@
         "\n",
         "mskctr_srcs.callback = CustomJS(\n",
         "    args=dict(\n",
+        "        all_tr_dtype_srcs=all_tr_dtype_srcs,\n",
         "        all_tr_shape_srcs=all_tr_shape_srcs,\n",
         "        all_tr_srcs=all_tr_srcs,\n",
         "        tr_srcs=tr_srcs\n",
         "    ), code=\"\"\"\n",
         "    var range = function(n){ return Array.from(Array(n).keys()); };\n",
         "\n",
         "    var inds = cb_obj.get('selected')['1d'].indices;\n",
-        "    var traces = all_tr_srcs.get('data')['traces'];\n",
+        "    var traces_dtype = all_tr_dtype_srcs.get('data')['traces_dtype'].constructor;\n",
```

Comment:
Get the `constructor` for our dummy value that has the same type that `traces` originally had. We can then leverage this to convert the `traces` from the decompressed `Uint8Array` to a `TypedArray` of the proper value type.

Note: This is a little bit hacky in that the `constructor` could be replaced by something strange. However, there is a fair argument that mucking with the `constructor` is what is hacky not accessing it.